templates/docs/patterns/grid/index.md

```diff
@@ -53,6 +53,16 @@ Read also: [Breakpoints](/docs/settings/breakpoint-settings)
     View example of the default grid
 </a></div>
 
+## 4-column grid
```

Comment:
Would make sense to mention "medium" screen size somewhere I think.

Reply:
Updated!